libs/transactions/src/lib/transactions-journey.module.ts

```diff
@@ -28,14 +33,18 @@ const defaultRoute: Route = {
   },
 };
 
+export interface TransactionsJourneyModuleConfig {
```

Comment:
I don't think I have seen it in the public API exports. Can you please check the PR via Microsoft API extractor, to make sure we have no forgotten exports?